### ISSUE TITLE
Mi Scale V1 sensor Error with 3.0.0

### ISSUE THREAD

**Issue by Kaibob2** [custom-components/ble_monitor#394]
After upgrading from 2.8.1 to 3.0.0 the following error appears in the log:

```
2021-06-07 11:37:03 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/ble_monitor/sensor.py", line 131, in async_run
    sensors = await async_add_sensor(mac, sensortype, firmware)
  File "/config/custom_components/ble_monitor/sensor.py", line 98, in async_add_sensor
    sensors.insert(device_sensors.index(sensor), globals()[SENSOR_DICT[sensor]](
KeyError: 'non-stabilized weight'
```

The ble_weight sensor for Mi Scale V1 becomes unavailable, too.

**Comment by Ernst79** [comment]
Ah, sorry, in const.py i've written `weight` with a capital `W`. Should be small. Will fix is a.s.a.p.

**Comment by Ernst79** [comment]
Fixed in 3.0.1, which has been released a minute ago.
